google_assistant/run.sh

```diff
@@ -1,27 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bashio
 set -e
 
-CONFIG_PATH=/data/options.json
 CLIENT_JSON=/data/client.json
 CRED_JSON=/data/cred.json
 
-CLIENT_SECRETS=$(jq --raw-output '.client_secrets' $CONFIG_PATH)
-PROJECT_ID=$(jq --raw-output '.project_id' $CONFIG_PATH)
-MODEL_ID=$(jq --raw-output '.model_id' $CONFIG_PATH)
+PROJECT_ID=$(bashio::config 'project_id')
+MODEL_ID=$(bashio::config 'model_id')
 
 # check if a new assistant file exists
-if [ -f "/share/$CLIENT_SECRETS" ]; then
-    echo "[Info] Install/Update service client_secrets file"
-    cp -f "/share/$CLIENT_SECRETS" "$CLIENT_JSON"
+if bashio::config 'client_secrets'; then
+    bashio::log:info "Install/Update service client_secrets file"
+    cp -f "$(bashio::config 'client_secrets')" "$CLIENT_JSON"
 fi
 
-if [ ! -f "$CRED_JSON" ] && [ -f "$CLIENT_JSON" ]; then
-    echo "[Info] Start WebUI for handling oauth2"
+if ! bashio::fs.directory_exists "${CRED_JSON}" && bashio::fs.directory_exists "${CLIENT_JSON}"; then
+    bashio::log:info "Start WebUI for handling oauth2"
     python3 /hassio_oauth.py "$CLIENT_JSON" "$CRED_JSON"
-elif [ ! -f "$CRED_JSON" ]; then
-    echo "[Error] You need initialize GoogleAssistant with a client secret json!"
+elif ! bashio::fs.directory_exists "${CRED_JSON}"; then
+    bashio::log:error "You need initialize GoogleAssistant with a client secret json!"
     exit 1
```

Comment:
these 2 lines can be replaced by a single `bashio::exit.nok "fatal error message to show"`